cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/service/BoardsConfigService.kt

```diff
@@ -30,11 +36,16 @@ class BoardsConfigService(
         contentRepository.save(widgetId, content)
     }
 
+    fun setDeletionHelper(helper: BoardDeletionHelper) {
+        boardDeletionHelper = helper
+    }
```

Comment:
I don't really like doing it via setter, but we really need it in one instance (I hope).

Reply:
so maybe this could be a constructor parameter with default value of `null` ?

Reply:
Moved to constructor.